specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -1964,6 +2310,10 @@
             "authnotrequired": {
               "type": "string",
               "description": "Specifies whether the authentication is disabled. Setting this property is highly discouraged from security point of view."
+            },
+            "aad-enabled": {
+              "type": "string",
+              "description": "Specifies whether aad based authentication has been enabled or disabled for the cache"
```

Comment:
>aad [](http://example.com/codeflow?start=48&length=3)

nit, I think AAD is the right capitalization for this brand

Reply:
Updated desc with AAD